docs/developers/celestia-app-vesting.md

```diff
@@ -0,0 +1,404 @@
+---
+sidebar_label: How to create a vesting account with Celestia App
```

Comment:
[not blocking] why can't I find this in the sidebar on PR preview?

I'm looking at https://docs.celestia.org/pr-preview/pr-841/developers/overview/

<img width="549" alt="Screenshot 2023-06-28 at 9 44 46 AM" src="https://github.com/celestiaorg/docs/assets/3699047/1c27adbd-a8d1-4b28-be15-291cf538beb1">

Reply:
^ disregard found it in https://docs.celestia.org/pr-preview/pr-841/developers/celestia-app-vesting/

after seeing the last file in this Github PR

Reply:
yea, it's hidden by default. do you think it would it be better if it is open by default?

Reply:
I don't have a strong opinion. TBH I wasn't expecting it to be under the "Wallet with Celestia App" category but idk where it belongs so defer to you.

Reply:
any ideas? i didn't want to bury it in resources, but it may fit better there

Reply:
expanding menu for now by default, i can't think of another way atm or better place